System/Random.hs

```diff
@@ -237,25 +238,13 @@ mutableByteArrayContentsCompat :: MutableByteArray s -> Ptr Word8
 {-# INLINE mutableByteArrayContentsCompat #-}
 
 -- $setup
+-- >>> import Control.Arrow (first, second)
+-- >>> import Control.Monad (replicateM)
+-- >>> import Data.Bits
+-- >>> import Data.Word
```

Comment:
With `-fobject-code`, it looks like imports from the outer scope are ignored, so the necessary ones need to be repeated inside a doctest block.